api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -34,12 +36,20 @@
     private static final EmailValidator VALIDATOR_EMAIL = EmailValidator.getInstance();
 
     @NotNull
+    @Autowired(required = false)
```

Comment:
since userDao has `@Inject` and there should only ever be one UserDao bean there would be no need to explicitly wire it in your XML config, but having it there doesn't hurt anything.